guides/assets/stylesheets/main.css

```diff
@@ -1,7 +1,7 @@
 /* Guides.rubyonrails.org */
 /* Main.css */
 /* Created January 30, 2009 */
-/* Modified February 8, 2009
+/* Modified July 3, 2020
```

Comment:
:rofl:  We've definitely [updated this](https://github.com/rails/rails/commits/master/guides/assets/stylesheets/main.css) since 2009-02-08!  I'm not sure, but perhaps these headers used to be auto-generated?  We can probably just remove them from this and the other CSS files in a separate PR.

Reply:
Ha -- I guessed as much, but wasn't sure. I'll revert this.